xtask/tests/server.Dockerfile

```
@@ -0,0 +1,40 @@
+# ---- Builder Stage ----
+FROM rust:1.88 as builder
+WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y wget iproute2 build-essential clang pkg-config git \
+<<<<<<< HEAD
+    && wget https://github.com/Kitware/CMake/releases/download/v3.30.0/cmake-3.30.0-linux-aarch64.tar.gz \
```

Hadolint (2.14.0)

[error] unexpected '&'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)